packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -2062,6 +2113,21 @@ class AnnotatedRegionLayer<T> extends ContainerLayer {
   /// Ignored if [size] is not set.
   final Offset offset;
 
+  /// Whether this layer should prevent layers visually behind it from receiving
+  /// the hit testing.
+  ///
+  /// If [opaque] is true, it will "absorb" the hit test if any of its children
+  /// absorb it, or if this layer contains the target position and has the
+  /// requested annotation. Absorbing the hit test prevents layers visually
```

Comment:
`layers` -> `sibling layers (or other layers that are not ancestors or descendants of this layer)`

Reply:
I changed it to "A hit that is absorbed will not be received by this layer's siblings (or any other layers that are not ancestors or descendants of this layer)."